polyphemus/lib/polyphemus/controllers/configuration_controller.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'controller'
+
+class ConfigurationController < Polyphemus::Controller
+  def action
+    return_data = {
+      magma: Polyphemus.instance.config(:magma)&.dig(:host),
```

Comment:
This will also let us add `metis_client` configuration values, which would make the user setup more minimal for Metis client.